### PR TITLE
[conf] allow qt5-qtmultimedia-plugin-audio-pulseaudio

### PR DESCRIPTION
**Pull request by d0b3rm4n**

### CODE REVIEW

File: allowed_libraries.conf [hunk 1]
```diff
@@ -35,6 +35,7 @@ libQt5Xml.so.5
 libQt5DBus.so.5
 libQt5WebKit.so.5
 libQt5Sensors.so.5
+libqtmedia_pulse.so
```
Comment (rburchell) [inline allowed_libraries.conf]: What's the impact of this? They shouldn't be linking directly to this as a library. It's a plugin used by QtMultimedia.

File: allowed_requires.conf [hunk 2]
```diff
@@ -57,6 +57,7 @@ qt5-qtgraphicaleffects
 
 # Qt Modules
 qt5-qtmultimedia
+qt5-qtmultimedia-plugin-audio-pulseaudio
```
Comment (rburchell) [inline allowed_requires.conf]: As a require though, it's fine.

File: allowed_libraries.conf (hunk 1 above)
Comment (d0b3rm4n) [inline allowed_libraries.conf]: ah ok, so nothing ever links against libqtmedia_pulse.so, then I can remove it from allowed_libraries.conf.

**Comment by rburchell** [comment]
lgtm

**Comment by thp** [comment]
LGTM, thanks. Note that due to the way the booster works, we'd also have to update this in `sailfish-api` (I can take care of that once this is merged).

**Pull request merged**